wps-cloudformation-template.yaml

```diff
@@ -57,7 +57,44 @@ Parameters:
   geoserverWpsEndpoint:
     Type: String
     Default: 'http://geoserver-wps.aodn.org.au/geoserver/wps'
+  environment:
+    Type: String
+    Default: dev
+    AllowedValues:
+      - production
+      - systest
+      - sandbox
+      - dev
+  createProdSystestBucket:
+    Type: String
+    Default: false
+    AllowedValues:
+      - true
+      - false
+Mappings:
+  EnvironmentMap:
+    production:
+      BucketName: "aws-wps-production"
```

Comment:
This could be templated instead of a map for each known environment... e.g.

```yaml
!Join [ "", [ "aws-wps-", !Ref environment ] ]
```

Reply:
The bucket names for the production will be fixed (something like aws-wps-production.aodn.org.au) and therefore I used mapping.